vendor/github.com/armPelionEdge/devicedb/Dockerfile.test_local

```diff
@@ -0,0 +1,5 @@
+FROM golang
```

Comment:
Might be good to specify a specific go version

Reply:
Resolving since this within `/vendor` and this is deprecated for a different testing method.